spring-boot-admin-samples/spring-boot-admin-sample-oauth2/spring-boot-admin-sample-oauth2-admin/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+server:
+  port: 8080
+
+logging:
+  level:
+    org.springframework.web.client.RestTemplate: TRACE
```

Comment:
Left on purpose. Since this is a demo application I thought it would be nice to have a log of the the request incoming from SBA server (including headers). Please let me know if you think it should me removed

Reply:
Please remove it. Imho setting this log level adds too much noise.

Reply:
Removed